TESTS/mbedmicro-rtos-mbed/basic/main.cpp

```diff
@@ -28,32 +28,36 @@
  * 6) More details on tests can be found in timing_drift_auto.py
  *
  */
-
 #include "mbed.h"
 #include "greentea-client/test_env.h"
+#include "unity.h"
+#include "utest.h"
 #include "rtos.h"
-#include "unity/unity.h"
 
 #if defined(MBED_RTOS_SINGLE_THREAD)
 #error [NOT_SUPPORTED] test not supported
 #endif
 
+using namespace utest::v1;
```

Comment:
Would it be possible to import the symbols needed rather than the whole namespace ?

Reply:
Generally I don't like importing anything, but rest of TEST code already use it. So I assumed that is prefered

If we don't want importing whole namespace we have to import following symbols:
```
using utest::v1::status_t;
using utest::v1::Case;
using utest::v1::Specification;
using utest::v1::greentea_test_setup_handler;
using utest::v1::greentea_test_teardown_handler;
using utest::v1::Harness;
```

Reply:
We do not have explicit rule on this item however, importing the minimal set of names used rather than the whole namespace may limit conflicts. We had some in the past with target code.